Gemfile.lock

```diff
@@ -113,6 +113,8 @@ GEM
     faraday (0.15.4)
       multipart-post (>= 1.2, < 3)
     ffi (1.11.1)
+    finite_machine (0.12.1)
```

Comment:
According to semantic versioning this isn't meant for production, yet?

Reply:
hmm... apparently it was good enough for Marks & Spencer :-)